.evergreen/run-tests.sh

```diff
@@ -4,7 +4,7 @@ set -o errexit  # Exit the script with error if any of the commands fail
 # Supported environment variables
 API_VERSION=${API_VERSION:-} # Optional API_VERSION environment variable for run-tests.php
 CRYPT_SHARED_LIB_PATH="${CRYPT_SHARED_LIB_PATH:-}" # Optional path to crypt_shared library
-MONGODB_URI=${MONGODB_URI:-} # Connection string (including credentials and topology info)
+MONGODB_URI=${MONGODB_URI:-}${APPEND_URI:-} # Connection string (including credentials and topology info)
```

Comment:
Nit: the "Supported environment variables" section here doubles as documentation, so I think it'd be clearer to take `APPEND_URI` separately in the same form and then merge them below.

That said, it's only essential that this merging happens _before_ the "Determine if MONGODB_URI already has a query string" logic below.

Reply:
Good point. I've moved the merging of `APPEND_URI` to before detecting the suffix.